llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -21355,11 +21355,11 @@ RISCVTargetLowering::EmitInstrWithCustomInserter(MachineInstr &MI,
            "ReadCounterWide is only to be used on riscv32");
     return emitReadCounterWidePseudo(MI, BB);
   case RISCV::Select_GPR_Using_CC_GPR:
-  case RISCV::Select_GPR_Using_CC_SImm5:
-  case RISCV::Select_GPR_Using_CC_SImm5NonZero:
-  case RISCV::Select_GPR_Using_CC_UImm5NonZero:
-  case RISCV::Select_GPR_Using_CC_SImm16NonZero:
-  case RISCV::Select_GPR_Using_CC_UImm16NonZero:
+  case RISCV::Select_GPR_Using_CV_CC_SImm5:
```

Comment:
The CV and QC are in a weird place. The pseudo name was a description "select a GPR using condition code". Now we have CV/QC in the middle.

Reply:
Really it should be at the beginning like all other vendor specific instructions, but I think that's hard to do with the way the multiclass is concatenating `NAME`.